MekHQ/src/mekhq/campaign/personnel/procreation/AbstractProcreation.java

```diff
@@ -466,7 +466,12 @@ public List<Person> birthHistoric(final Campaign campaign, final LocalDate today
             baby.setLoyalty(Compute.d6(3) + 2);
 
             // set education based on age
-            baby.setEduHighestEducation(EducationLevel.EARLY_CHILDHOOD);
+            if(baby.getAge(today) < 16) {
+                baby.setEduHighestEducation(EducationLevel.EARLY_CHILDHOOD);
+            }
```

Comment:
Very nit-picky, but can the `else` be on the same line as the `}` as that's how we format it across the codebase

Reply:
Yep, done, sorry I missed that it's even in the style guide.